src/App.css

```diff
@@ -1,3 +1,53 @@
 body {
+  background-color: #fafafa;
+}
+
+nav {
+  display: flex;
+  flex-direction: row;
+  padding-left: 20px;
+  color: #0893ff;
+  font-weight: 500;
+  background-color: #fff;
+}
+
+.nav-links {
+  display: flex;
+  align-items: center;
+  color: black;
+  font-weight: 300;
+  font-size: 120%;
+}
+
+.nav-links > * {
+  margin-left: 10px;
+  color: black;
+  text-decoration: none;
+}
```

Comment:
- [ ] It would be better to follow the design, using the correct `padding, margin` properties to match the template.
- [ ] You should use the 2 font families from the design that are: `Montserrat` `RobotoSlab`, and check which text are using each font family.
- [ ] Please, don't forget to add the user icon. It doesn't have any functionality.

**Yours**
![image](https://user-images.githubusercontent.com/79876959/145666315-cff5bdbc-6c0b-41a2-a30c-4292ce97e67f.png)


**Template**
![image](https://user-images.githubusercontent.com/79876959/145666300-f7025242-f66d-4a54-a746-12a37f7f8881.png)